src/app.js

```diff
@@ -77,7 +77,7 @@ export default function (app) {
 
           } else {
             for (i = 0; i < hooks.onUpdate.length; i++) {
-              hooks.onUpdate[i](model, result, data)
+              hooks.onUpdate[i](model, result, data, actions)
```

Comment:
Thinking about this now.. I think `(oldModel, model, data, actions)` actually makes sense, see my last commit and comment